### DISCUSSION TITLE
What is the red line in Timesheet?

### DISCUSSION THREAD

**Discussion by WhiteTiger-IT** [kimai/kimai#2337] [General]
A dotted red line appeared on the Timesheet.
What does it indicate?

![Kimai2 RedLine](https://user-images.githubusercontent.com/20139133/107376091-c6ca4100-6ae9-11eb-9d0b-75b63ed60c2a.png)

**Comment by kevinpapst** [comment]
An overlapping entry: a timespan that is (partially) recorded twice - see https://github.com/kevinpapst/kimai2/pull/1534

**Reply by WhiteTiger-IT** [reply]
Yes, there is an overlap of schedules because the two jobs overlapped.
OK, thanks.
